app/models/stats/management_information_generator.rb

```diff
@@ -24,8 +24,10 @@ def generate_new_report
         csv << Settings.claim_csv_headers.map { |header| header.to_s.humanize }
         Claim::BaseClaim.active.non_draft.find_each do |claim|
           ClaimCsvPresenter.new(claim, 'view').present! do |claim_journeys|
-            claim_journeys.each do |claim_journey|
-              csv << claim_journey
+            if claim_journeys.any?
+              claim_journeys.each do |claim_journey|
```

Comment:
**[nit pick/not a blocker]** If `claim_journeys` is an empty array I believe the `claim_journeys.any?` is irrelevant as the `.each` with immediately end. The only case where I can see this failing would be if `claim_journeys` was `nil`, but in that case even `.any?` would fail.

What are we trying to achieve with that extra check (I might lack context :P )?

Reply:
Good points...
Claim journeys is returning `[nil]` but the `if` code block could be leftover from a previous iteration... I'll test and update

Reply:
I would probably just use `claim_journeys.compact` then if that is the case, but again, no blockers, your solution works as well ;)